### ISSUE TITLE
PicoRV32 example?

### ISSUE THREAD

**Issue by sunnystormy** [mystorm-org/BlackIce-II#26]
Hello! I've still got my BlackIce-II board and am trying to get Clifford Wolf's PicoRV32 running on it. I'm running into some problems with getting next-pnr to create an "hx4k" pathway (I know that the tutorials recommend arachne-pnr, but that seems to have been deprecated).

If you have any recommendations for getting a risc-v core running on the board, I'd love to hear it! Thank you for your help!

**Comment by lawrie** [comment]
--hx8k --package tq144:4k

works for me.

**Comment by sunnystormy** [comment]
So, I've modified the Makefile that comes with picorv32 to include the options you've provided, and I end up with:

```
nextpnr-ice40 --hx8k --package tq144:4k --json synth.json --pcf example.pcf --asc example.asc
Info: Importing module top
Info: Rule checker, verifying imported design
Info: Checksum: 0x6c59d296

ERROR: package does not have a pin named 'J3' (on line 1)
ERROR: Loading PCF failed.
0 warnings, 2 errors
make: *** [Makefile:35: example.asc] Error 255
```

Any idea why this is happening? Is it because the pcf file may have the wrong pin configuration?

Thank you for your help!

**Comment by lawrie** [comment]
Yes, you were using the wrong pcf file for the Blackice II board.

**Comment by sunnystormy** [comment]
@lawrie Thank you for this! I'll move the conversation to a more appropriate venue, as per your suggestion.
